Makefile

```diff
@@ -97,8 +97,13 @@ validate: lint
 	@BUILDTAGS="$(BUILDTAGS)" hack/validate.sh
 
 .PHONY: lint
+lint: LINTER_TAGS = containers_image_openpgp containers_image_docker_daemon_stub containers_image_storage_stub containers_image_fulcio_stub containers_image_rekor_stub
```

Comment:
Can you sort the tags alphabetically, please?

Reply:
Done